src/Compilers/Core/Portable/DocumentationCommentId.cs

```diff
@@ -344,21 +374,59 @@ public override void VisitMethod(IMethodSymbol symbol)
             public override void VisitNamespace(INamespaceSymbol symbol)
             {
                 _builder.Append("N:");
-                _generator.Visit(symbol);
+                AppendNamespace(symbol, _builder);
```

Comment:
>AppendNamespace(symbol, _builder); [](http://example.com/codeflow?start=16&length=34)

This change feels outside of scope of this PR and, I think,  should be excluded. I suggest opening a dedicated issue, and, optionally, creating a separate PR with appropriate tests added (those that fail without the change).